flow360/component/flow360_params/solvers.py

```diff
@@ -87,7 +87,7 @@ class PressureCorrectionSolver(Flow360BaseModel):
     """:class:`PressureCorrectionSolver` class"""
 
     randomizer: LinearIterationsRandomizer = pd.Field()
-    linear_solver: LinearSolver = pd.Field(alias="linearSolver", default=LinearSolver())
+    linear_solver: LinearSolver = pd.Field(alias="linearSolver")
```

Comment:
why remove default here?

Reply:
Requested change from frontend, see https://y2x1d18d8u.larksuite.com/docx/Sgo7dgrdKoGTj3xXiWOu5R7jsmh?from=from_copylink in "remove defaults from parent" part

Reply:
Added it back, deleting duplicate defaults is now handled as a schema postprocessing step